lib/PuppeteerSharp/Messaging/PageConsoleResponse.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+
+namespace PuppeteerSharp.Messaging
+{
+    public class PageConsoleResponse
+    {
+        [JsonProperty("type")]
+        public string Type { get; set; }
```

Comment:
`Newtonsoft.Json` knows how to deserialize enums (`StringEnumConverter`) so we can use `ConsoleType` here and remove the conversion in the 
`ConsoleMessage` class

Reply:
@Meir017 I got "inspiration" from Dialog.cs made by @kblok , which uses a mapping method but you are right. @kblok what do you think ?

Reply:
If you can also improve Dialog go for it :)

Reply:
It only works when deserializing since:
```cs
JsonConvert.SerializeObject(DialogType.BeforeUnload)
// output: "\"beforeUnload\"" 
```
and not `beforeunload`

Reply:
A possible solution is to use `[EnumMember(Value = "beforeunload")]` for enum values where the default cameCase doesn't work, does this make sense?

Reply:
I made the changes, but didn't include "[JsonConverter(typeof(StringEnumConverter), true)]", do you confirm it's necessary ? It works find without it to deserialize, do we need to serialize these values too ?

Reply:
If we don't really need it, don't add it.